### PR TITLE
nextra docs

### PR DESCRIPTION
**Pull request by danstarns**
Moving graphql-debugger.com to a site built with [nextra](https://nextra.site/) and overhauling content on the way. 

https://preview.graphql-debugger.com

### CODE REVIEW

File: docs/src/pages/docs/components/adapters.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: docs
+category: Documentation
+permalink: /docs/components/adapters
+next: /docs/components/adapters
```
Comment (dimaMachina) [inline docs/src/pages/docs/components/adapters.mdx:5]: these frontMatter props are from graphql.org, you can remove them

File: docs/src/pages/docs/why.mdx [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+        user { ## <----- SELECT * FROM users
+          name
+          email
+        }
+}
+```
+    </pre>
+  </Tabs.Tab>
+  <Tabs.Tab>
+    <pre>
```
Comment (dimaMachina) [inline docs/src/pages/docs/why.mdx:82]: you don't need pre tag

File: pnpm-workspace.yaml [hunk 3]
```diff
@@ -3,3 +3,4 @@ packages:
   - "apps/**"
   - "e2e/**"
   - "plugins/**"
+  - "docs/**"
```
Comment (dimaMachina) [inline pnpm-workspace.yaml:6]:
```suggestion
  - docs
```

File: docs/src/pages/docs/_meta.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+export default {
+  "-- 1": {
+    type: "separator",
+    title: "Docs",
+  },
```
Comment (dimaMachina) [inline docs/src/pages/docs/_meta.ts:5]: can be removed since you don't have several sections

File: docs/src/pages/_meta.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    type: "page",
+    display: "hidden",
+  },
+  docs: {
+    type: "page",
+    title: "Documentation",
+  },
+  opensource: {
+    type: "page",
+    title: "Github ↗",
```
Comment (dimaMachina) [inline docs/src/pages/_meta.ts:12]:
```suggestion
    title: "GitHub ↗",
```

File: docs/src/pages/docs/_meta.ts (hunk 4 above)
Comment (danstarns) [reply docs/src/pages/docs/_meta.ts:5]: Thanks - removed was wondering what this was!

File: docs/src/pages/docs/components/adapters.mdx (hunk 1 above)
Comment (danstarns) [reply docs/src/pages/docs/components/adapters.mdx:5]: Should we include them for SEO or any other related reason? Is it the whole block that we dont need?

File: docs/src/pages/docs/why.mdx (hunk 2 above)
Comment (danstarns) [reply docs/src/pages/docs/why.mdx:82]: Thanks, I thought it was helping with the formatting. 

I am experiencing a slight issue where I have to indent the code blocks inside the tabs very far. 

```
  <Tabs.Tab>
      \`\`\`graphql
query {
        user { ## <----- SELECT * FROM users
          name
          email
          posts { ## <----- SELECT * FROM posts
            title
            body
          }
        }
}
\`\`\`
  </Tabs.Tab>  
```

Is there a way I dont need to do this?

Comment (danstarns) [reply docs/src/pages/docs/why.mdx]: Screenshot 2024-01-20 at 10 24 43
Screenshot 2024-01-20 at 10 25 00

Comment (danstarns) [reply docs/src/pages/docs/why.mdx]: * https://github.com/rocket-connect/graphql-debugger/pull/206/commits/8457911296c3e867bee0f0628dbdd5f55da0d131

File: docs/src/pages/docs/components/adapters.mdx (hunk 1 above)
Comment (dimaMachina) [reply docs/src/pages/docs/components/adapters.mdx:5]: The whole block was used by previous graphql.org

Comment (danstarns) [reply docs/src/pages/docs/components/adapters.mdx]: Got it, they are gone - thanks

* https://github.com/rocket-connect/graphql-debugger/pull/206/commits/2ea041b38f00487cbce39d308ca7c6ca46261e6b

File: docs/src/pages/docs/components/adapters.mdx [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+## Proxy Adapter
+
+The Proxy Adapter will forward all read and writes over the [GraphQL Schema](/docs/packages/graphql-schema) and to the [Collector](/docs/components/collector). This is what most will use when they want to use GraphQL Debugger, if it may be that you are pointing to your local or remote instance of GraphQL Debugger.
+
+### Getting Started
+
+#### Install
+
+```bash
```
Comment (dimaMachina) [inline docs/src/pages/docs/components/adapters.mdx:17]:
```suggestion
```bash npm2yarn
```
will generate tabs for all package managers

File: docs/src/pages/docs/components/adapters.mdx [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+## SQLite Adapter
+
+This is used inside the free dev tooling GraphQL Debugger provides. It is a simple SQLite database that stores all the data locally. This is not recommended for production use.
+
+### Getting Started
+
+#### Install
+
+```bash
```
Comment (dimaMachina) [inline docs/src/pages/docs/components/adapters.mdx:43]:
```suggestion
```bash npm2yarn
```

File: docs/src/pages/docs/components/adapters.mdx (hunk 6 above)
Comment (danstarns) [reply docs/src/pages/docs/components/adapters.mdx:17]: @dimaMachina can we exclude `bun` ?

**Comment by danstarns** [comment]
* https://github.com/rocket-connect/graphql-debugger/pull/212

File: docs/src/pages/_document.tsx [hunk 8]
```diff
@@ -1,9 +1,12 @@
+import { GoogleAnalytics } from "@next/third-parties/google";
 import { Head, Html, Main, NextScript } from "next/document";
 
 export default function Document() {
   return (
     <Html lang="en">
-      <Head />
+      <Head>
+        <GoogleAnalytics gaId="G-KWK6N3X1ZD" />
```
Comment (danstarns) [inline docs/src/pages/_document.tsx:8]: @dimaMachina is this the recommended way to put google analytics on our site?

**Review by danstarns — COMMENTED** [review]
👀

File: docs/src/pages/_document.tsx (hunk 8 above)
Comment (dimaMachina) [reply docs/src/pages/_document.tsx:8]: Use nex.js recommendations

File: docs/src/pages/docs/components/adapters.mdx (hunk 6 above)
Comment (dimaMachina) [reply docs/src/pages/docs/components/adapters.mdx:19]: not for the moment

**Pull request merged**